App.js

```diff
@@ -49,9 +50,15 @@ export default function App() {
     Poppins_900Black_Italic,
   });
 
-  if (!fontsLoaded) {
-    return <AppLoading />;
-  }
+  useEffect(() => {
+    async function prapare() {
+      await SplashScreen.preventAutoHideAsync();
+    }
+    prapare();
+  }, []);
+
+  if (!fontsLoaded) return undefined;
+  else SplashScreen.hideAsync();
```

Comment:
`else` clause won't be needed here in my humble opinion. `SplashScreen.hideAsync();` would be enough.

Reply:
hmmm... I just removed it.